translator/internaldata/oc_to_traces.go

```diff
@@ -150,16 +150,16 @@ func ocSpanToInternal(dest data.Span, src *octrace.Span) {
 	dest.SetDroppedEventsCount(droppedEventCount)
 	dest.SetLinks(links)
 	dest.SetDroppedLinksCount(droppedLinkCount)
-	ocStatusToInternal(src.Status, dest)
+	ocStatusToInternal(dest.Status(), src.Status)
 }
 
-func ocStatusToInternal(ocStatus *octrace.Status, out data.Span) {
+func ocStatusToInternal(dest data.SpanStatus, ocStatus *octrace.Status) {
```

Comment:
This uses a different order of parameters than e.g. ocSpanToResourceSpans. Let's decide on one ordering and stick with it.

I think majority of functions use (input,output) ordering. We can also do (output,input), it does not matter, but let's make sure it is consistently used everywhere by all translation functions.

Reply:
Done. went with (src, dest) pattern everywhere.